node/node.d.ts

```diff
@@ -2299,6 +2299,18 @@ declare module "util" {
     export function isError(object: any): boolean;
     export function inherits(constructor: any, superConstructor: any): void;
     export function debuglog(key: string): (msg: string, ...param: any[]) => void;
+    export function isBoolean(object: any): boolean;
```

Comment:
you can use `export function isBoolean(object: any): object is boolean;`
https://github.com/Microsoft/TypeScript/issues/5764
http://goo.gl/Dhd7eS